src/views/Dashboard.vue

```diff
@@ -1,18 +1,15 @@
 <template>
-  <v-container
-    fluid
-    grid-list
-    class="c-dashboard mt-4"
-  >
+  <v-container fluid grid-list class="c-dashboard mt-4">
```

Comment:
I think there's no change on this line, just formatting, could you undo the formatting changes, please? This way we can compare this diff again in the future when upgrading Vuetify again. So if we can have the bare minimum of what was required to upgrade from 2.0 to 2.1, that'd be helpful :+1: